detail.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
```

Comment:
get rid of unnecessary imports that aren't actually being used in this file

Reply:
That's perfectly correct. Thank you!